src/components/Profile/ProfileInfo.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Image from 'next/image';
+import {
+  setResearchPercent,
+  setServicePercent,
+  setTeachingPercent,
+  reset,
+  selectTeachingPercent,
+  selectResearchPercent,
+  selectServicePercent,
+} from '@/store/profile.store';
+import PercentageInfo from './PercentageInfo';
+import { UpdateProfessorInfoDto } from '@/models/professorInfo.model';
+import { updateProfessorInfoForUser } from '@/client/professorInfo.client';
+import { ResponseStatus } from '@/client/activities.client';
+import { useRouter } from 'next/router';
+
+export interface ProfileInformation {
+  firstName: string;
+  lastName: string;
+  email: string;
+  position: string;
+  teachingPercent: number;
+  researchPercent: number;
+  servicePercent: number;
+}
+
+type ProfileInfoProps = ProfileInformation & {};
```

Comment:
Q:what's this do?

Reply:
I used this weird type intersection (&) because initially the ProfileInfoProps was going to have an addition field `editing`, so that line looked like `type ProfileInfoProps = ProfileInformation & { editing: boolean }`. It's a typescript syntax for the intersection of two interfaces. I'll change this to be just `type ProfileInfoProps = ProfileInformation`.